Dockerfile.travis

```diff
@@ -28,9 +28,10 @@ WORKDIR /src/healthcare-application
 # Ensure any deltas in package.json are reflected.
 COPY package.json package.json
 COPY npm-shrinkwrap.json npm-shrinkwrap.json
+COPY webpack.config.js webpack.config.js
```

Comment:
This isn't quite right.

Node creates a layered filesystem for each command run.  You want to have the lowest amount of variance in the target filesystem so that it can get cached.

Putting this line above the `RUN npm install` line means that Docker build will not be able to reuse the node_modules of a previous build if webpack.config.js changes.

I recommend removing this line and putting the run after the `ADD . .`

That's probably the "right" way to do things.